brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkFilter.java

```diff
@@ -0,0 +1,77 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.*;
+import com.github.kristofa.brave.http.DefaultSpanNameProvider;
+import com.github.kristofa.brave.http.HttpServerRequestAdapter;
+import com.github.kristofa.brave.http.HttpServerResponseAdapter;
+import com.github.kristofa.brave.http.SpanNameProvider;
+import com.github.kristofa.brave.spark.internal.MaybeAddClientAddressFromRequest;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveSparkFilter {

```

Comment:
I'd rename this type to `BraveTracing` as it flows nicer, also helps as not everyone knows brave is for tracing.. plus #350 is going for this convention.

Ex.

```java
BraveTracing tracing = BraveTracing.create(brave);
Spark.before(tracing.before());
Spark.exception(Exception.class, tracing.exception(new ExceptionHandlerImpl()));
Spark.afterAfter(tracing.afterAfter());
```